### PR TITLE
Adding https file

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

PR for https://github.com/eclipse/codewind/issues/269.

This PR is ready for review.

### CODE REVIEW

**Comment by sishida** [comment]
@rajivnathan and @maysunfaisal, This PR is ready for review! Please let me know if it looks good to go if you would like me to make other changes. Thanks!

**Comment by maysunfaisal** [comment]
@sishida On further thought, maybe we should add a warning saying, this is only an example for development and should not be used for a production application.

**Comment by sishida** [comment]
@maysunfaisal OK, I just want to make sure I understand correctly...We want to make sure that users don't copy/paste this example code into the applications that they create themselves?

If they do attempt to use one of these examples in their applications, will the code not work as intended? 

Could we give them an example that they could apply to their own applications, or is the code dependent on individual applications (making a one-size-fits-all-apps option basically impossible)?

**Comment by maysunfaisal** [comment]
@sishida 

They can apply and use these examples to their use case. And the changes are dependent on applications like how I mentioned only two ways ie Microprofile and Python(Other types may/will have other steps).

But it is a self signed certificate and the steps mentioned and outlined here, may not be compliant with their security policy/usage. This is just a basic getting started sort of example. They should probably be more stringent on their final production application.

**Comment by sishida** [comment]
@maysunfaisal I have added a "Caution" note to my latest commit.

@rajivnathan Could you also please review this new file when you get the chance?

Thanks, team!

**Comment by maysunfaisal** [comment]
sounds good

File: docs/_documentations/convertingtohttps.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ type: document
 
 You can convert Codewind applications from HTTP to HTTPS. These examples feature a Python and a MicroProfile application.
 
+**Caution:** Use these examples for development but not for production application. The self-signed certificate and steps shown here might not comply with the security policy and usage policy of your project.
```
Comment (rajivnathan) [inline docs/_documentations/convertingtohttps.md]: Should it be `production applications` (plural)?

**Comment by sishida** [comment]
@rajivnathan I was wondering that myself--At first I wasn't sure if the term was supposed to singular...but it does make sense in the plural. :) I have changed it in the latest commit.

**Comment by sishida** [comment]
@elsony Could you please review this PR?

**Comment by sishida** [comment]
@elsony I've also added "HTTPS application" info with this PR:https: //github.com/eclipse/codewind-docs/pull/133/files

Still need to add the equivalent info for VS Code.

**Comment by maysunfaisal** [comment]
@elsony like @sishida mentioned, we have that information along with the other project settings in the link Sarah posted, this is just a gettting started example on how to convert a HTTP to HTTPS

**Comment by sishida** [comment]
@elsony Do you still want us to include this example file, or do you think the content in this PR (https://github.com/eclipse/codewind-docs/pull/133/files) will suffice? Please let me know. Thank you! 😄

**Comment by elsony** [comment]
@sishida After talking to @maysunfaisal , although this is a useful piece of information, this is just example on how to enable https and those steps are not specific to the templates that we provide.  Therefore, I suggest to turn this to an article instead of putting that as part of the Codewind documentation.  When we put that to an article, we should also add the step for changing the https settings in Codewind to tie that back to Codewind instead.

**Comment by sishida** [comment]
@elsony Thank you for your input. I will close this PR and open another issue for adding this PR content as an article instead. 👍

**Pull request closed without merging**